Source/Core/Core/DSP/Jit/x64/DSPJitUtil.cpp

```diff
@@ -264,7 +264,7 @@ void DSPEmitter::dsp_op_read_reg(int reg, Gen::X64Reg host_dreg, RegisterExtensi
     CMP(64, R(host_dreg), acc_reg);
     FixupBranch no_saturate = J_CC(CC_Z);
 
-    CMP(64, acc_reg, Imm32(0));
+    CMP_or_TEST(64, acc_reg, Imm32(0));
```

Comment:
You could just write `TEST(64, acc_reg, acc_reg);` directly, since `CMP_or_TEST` is the type of thing for when the arguments are only known at jit-time.

Reply:
I got confused when looking at the DSPJitRegCache code, so I erred on the side of caution. Thanks for pointing that out to me, I've changed this now.